scripts/start/prepare_and_run.R

```diff
@@ -471,7 +471,10 @@ prepare <- function() {
   
   # copy right gdx file to the output folder
   gdx_name <- paste0("config/gdx-files/",cfg$gms$cm_CES_configuration,".gdx")
-  system(paste0('cp ',gdx_name,' ',file.path(cfg$results_folder, "input.gdx")))
+  if (0 != system(paste('cp', gdx_name, 
+			file.path(cfg$results_folder, 'input.dx')))) {
```

Comment:
shouldn't this be 'input.gdx', not 'input.dx'?

Reply:
Sure should.